### DISCUSSION TITLE
how to add two input parameters in Zingg argument.

### DISCUSSION THREAD

**Discussion by iitbhumanish** [zinggAI/zingg#587] [Q&A]
I tried to add both input pipe but at the end only one input pipe is shown in args. Please advise me if missing something.

![image](https://user-images.githubusercontent.com/131442577/236833060-0fcf226b-358e-4b32-8901-5d089e93b9bf.png)

![image](https://user-images.githubusercontent.com/131442577/236833124-198a75af-5314-4353-9a2a-bc467f252fdc.png)

**Comment by sonalgoyal** [comment]
thanks for opening a new thread @iitbhumanish. The setData method takes an array.

https://zingg.readthedocs.io/en/0.3.4/zingg.html#zingg.client.Arguments.setData 
https://github.com/zinggAI/zingg/blob/main/examples/amazon-google/AmazonGoogle.py

**Comment by iitbhumanish** [comment]
Thanks a  lot . it worked.
